src/test/resources/org/broadinstitute/hellbender/utils/WDLTestExpectedOutputs/expectedTestWDLToolAllArgsInputs.json

```diff
@@ -1,5 +1,5 @@
 {
-  "TestWDLTool.dockerImage": "broadinstitute/gatk:1.2.3.4",
+  "TestWDLTool.dockerImage": "broadinstitute/gatk:4.2.4.0",
```

Comment:
@aofarrel The files in `src/test/resources/org/broadinstitute/hellbender/utils/WDLTestExpectedOutputs` are not used with real WDLs; they're expected output files used to test GATK's WDL autogeneration, and use dummy/non existent GATK versions since we don't want the expected outputs to vary for each build number. Those values need to remain  constant - so just revert your changes to the 3 files in that folder and those failures should go away.

Reply:
Oh, that explains it! I was wondering why they all were numbered 1234. Fixed and seems to be passing now.